### PR TITLE
Move the Keyboardio Model01 to a keyboardio/ subdir

### PR DESCRIPTION
**Pull request by algernon**
## Description

As there are more boards from Keyboardio in the works, in preparation for supporting them in QMK too, move the Model01 to `keyboardio/model01` instead of having it at the top level.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [x] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/keyboardio/model01/readme.md [hunk 1]
```diff
@@ -2,13 +2,13 @@
 
 A split keyboard.
 
-Keyboard Maintainer: QMK Community  
-Hardware Supported: Keyboardio Model 01  
-Hardware Availability: [Keyboardio](https://shop.keyboard.io)  
+Keyboard Maintainer: QMK Community
+Hardware Supported: Keyboardio Model 01
+Hardware Availability: [Keyboardio](https://shop.keyboard.io)
```
Comment (fauxpark) [inline keyboards/keyboardio/model01/readme.md]:
```suggestion
* Keyboard Maintainer: QMK Community
* Hardware Supported: Keyboardio Model 01
* Hardware Availability: [Keyboardio](https://shop.keyboard.io)
```

File: keyboards/keyboardio/readme.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+**Website:** https://shop.keyboard.io/
+
+## Social Media
+
+**Discord:** https://discord.gg/4az77sf
+**Forums:** https://community.keyboard.io/
+**Twitter:** https://twitter.com/keyboardio
+**Facebook:** https://facebook.com/keyboardio
+**Blog:** https://blog.keyboard.io/
```
Comment (fauxpark) [inline keyboards/keyboardio/readme.md]:
```suggestion
**Discord:** https://discord.gg/4az77sf  
**Forums:** https://community.keyboard.io/  
**Twitter:** https://twitter.com/keyboardio  
**Facebook:** https://facebook.com/keyboardio  
**Blog:** https://blog.keyboard.io/
```

File: keyboards/keyboardio/model01/readme.md [hunk 3]
```diff
@@ -17,7 +17,7 @@ Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/ne
 
 To program the keyboard, if you have avrdude installed:
 
-    make model01:default:avrdude
+    make keyboardio/model01:default:avrdude
```
Comment (drashna) [inline keyboards/keyboardio/model01/readme.md]: This is a minor nitpick, but: 

```suggestion
    make keyboardio/model01:default:flash
```

**Comment by noroadsleft** [comment]
Merging this through the `future` branch; will write a changelog for it later (there's a few of these PRs in this queue).

Thanks!

**Pull request merged**